x-pack/plugins/security_solution/public/detections/components/rules/throttle_select_field/index.tsx

```diff
@@ -13,14 +13,18 @@ import {
 } from '../../../../../common/constants';
 import { SelectField } from '../../../../shared_imports';
 
-export const THROTTLE_OPTIONS = [
-  { value: NOTIFICATION_THROTTLE_NO_ACTIONS, text: 'Perform no actions' },
+export const THROTTLE_OPTIONS_FOR_BULK_RULE_ACTIONS = [
   { value: NOTIFICATION_THROTTLE_RULE, text: 'On each rule execution' },
   { value: '1h', text: 'Hourly' },
   { value: '1d', text: 'Daily' },
   { value: '7d', text: 'Weekly' },
 ];
 
+export const THROTTLE_OPTIONS = [
```

Comment:
Nit: maybe `THROTTLE_OPTIONS_FOR_RULE_CREATION_AND_EDITING` would be more descriptive